src/main/java/com/zufar/icedlatte/filter/product_filter/ProductFilterMinPrice.java

```diff
@@ -0,0 +1,23 @@
+package com.zufar.icedlatte.filter.product_filter;
```

Comment:
I think that all this logic should be located in `com.zufar.icedlatte.product.filter` package. 

Also, please note that the snake_case is not common in Java